active/operators/bort/generate_job_id.py

```diff
@@ -10,7 +10,7 @@ def generate_job_id(**kwargs):
     """
     last_run_id = _get_last_bort_run_id()
     #TODO implement _get_last_bort_update_date()
-    last_bort_update_date = _get_last_bort_update_date()
+    last_bort_update_date = _get_last_resynth_update_date()
```

Comment:
this is kind of confusing naming.
we're going to have both a last_bort_update_date as well as a _last_ner_update_date right?
so we can run only notes which DO have an ner_update_date but which DO NOT have a bort_update_date

Reply:
disregard - I just saw the change in the other commit